tap.js

```diff
@@ -37,8 +59,9 @@
             this.el.addEventListener('touchend', this, false);
             this.el.addEventListener('touchcancel', this, false);
 
-        } else if (e.type === 'mousedown') {
+        } else if (e.type === 'mousedown' && this.whichbutton(e) === 1) {
```

Comment:
Yep, I think this makes sense. Can you please remove the cruft for all the other mouse buttons that we don't need? Then I think this should be good :+1: